test/unit/uk/gov/hmrc/exports/controllers/CancellationControllerSpec.scala

```diff
@@ -57,46 +57,42 @@ class CancellationControllerSpec extends UnitSpec with GuiceOneAppPerSuite with
     "return 200" when {
       "request is valid" in {
         withAuthorizedUser()
-        given(submissionService.cancel(any(), any())(any())).willReturn(Future.successful(CancellationRequested))
+        given(submissionService.cancel(any(), any())(any())).willReturn(Future.successful(CancellationRequestSent))
 
         val result: Future[Result] = route(app, post.withJsonBody(toJson(cancellation))).get
 
         status(result) mustBe OK
-        contentAsString(result) mustBe empty
+        contentAsJson(result) mustBe Json.toJson(CancellationRequestSent.toString)
```

Comment:
Could not be `contentAsString(result) mustBe CancellationRequestSent.toString` ?

Reply:
No that does not work as its JSON so wraps the value in quotes. This would work but I think its uglier than the above:
`contentAsString(result) mustBe s""""${CancellationRequestSent.toString}""""`